pkg/home/manager.go

```diff
@@ -55,10 +54,10 @@ var (
 
 func Initialize() error {
 	builder := types.BuilderTypeDocker
-	dockerVersion, err := docker.GetDockerVersion()
-	if err == nil && dockerVersion > 22 {
-		builder = types.BuilderTypeMoby
-	}
+	// dockerVersion, err := docker.GetDockerVersion()
+	// if err == nil && dockerVersion > 22 {
+	// 	builder = types.BuilderTypeMoby
+	// }
```

Comment:
```suggestion
```

Delete the legacy code. Can recover from the previous commit.

Reply:
fixed